### ISSUE TITLE
Unsupported operation: _Namespace (web)

### ISSUE THREAD

**Issue by solomora** [miguelpruivo/flutter_file_picker#556]
Hello, I want to create a File object (from dart:io), to be able to access the .readAsString() method in Flutter Web
I am using File.fromRawPath (platformFile.bytes) to instantiate it but I get the title error and this console output from here below

**Console:**
```
Report.pdf
Error: Unsupported operation: _Namespace
    at Object.throw_ [as throw] (http://localhost:51572/dart_sdk.js:5280:11)
    at Function.get _namespacePointer [as _namespacePointer] (http://localhost:51572/dart_sdk.js:54662:17)
    at Function._namespacePointer (http://localhost:51572/dart_sdk.js:52343:28)
    at Function._dispatchWithNamespace (http://localhost:51572/dart_sdk.js:52348:31)
    at io._File.fromRawPath.open (http://localhost:51572/dart_sdk.js:52497:23)
    at io._File.fromRawPath.readAsBytes (http://localhost:51572/dart_sdk.js:52669:19)
    at io._File.fromRawPath.readAsString (http://localhost:51572/dart_sdk.js:52717:19)
    at getFile (http://localhost:51572/packages/dragon_y_tintometro/src/ui/home_page.dart.lib.js:426:37)
    at getFile.next (<anonymous>)
```
**Code:**
```
Future<String> getContent() async {
    String content;
    FilePickerResult result = await FilePicker.platform.pickFiles(
      allowMultiple: false,
      type: FileType.custom,
      allowedExtensions: ['pdf'],
    );
    if (result != null) {
      final PlatformFile platformFile = result.files.first;
      print(platformFile.name);

      final File file = File.fromRawPath(platformFile.bytes);

      content = await file.readAsString();
      print(content);
    }
    return content;
  }
```

the web app stops here:
![file_picker_web](https://user-images.githubusercontent.com/70462879/102828470-91478800-43c3-11eb-8dc4-5a57abb70984.png)

**Comment by miguelpruivo** [comment]
Hi. On web you need to access `bytes` because paths aren’t supported by browsers.

You have an example on how to use it [here](https://github.com/miguelpruivo/flutter_file_picker/wiki/FAQ).

**Comment by solomora** [comment]
Hello, thanks for the information, in the same way .. I need to access the information inside the pdf file, that is why I want to instantiate the File object, having only the bytes is not enough for me

**Comment by solomora** [comment]
> Hi. On web you need to access `bytes` because paths aren’t supported by browsers.
> 
> You have an example on how to use it [here](https://github.com/miguelpruivo/flutter_file_picker/wiki/FAQ).

I am using the bytes of the selected file to instantiate with the .fromRawPath() method of the File class of dart: io.
I need to use the File.readAsString () method to read the content in the selected pdf file

**Comment by summerlab** [comment]
@solomora  do you find the way？ can u show me your code？ thks

**Comment by solomora** [comment]
Hi, unfortunately no, i abandoned the project, maybe in this release of
flutter (2.2) you can run this on web. Good Luck!.

**Comment by himmatsingh019** [comment]
Hey, I want to get the size of the file/image for flutter web. I am using image_size_getter: package, but it is throwing Unsupported operation: _Namespace error. Can anyone help me regarding this.

**Comment by hamadbintariq4u** [comment]
plz anybody say me what i do

**Comment by hyungtaecf** [comment]
I'm using the package universal_io to make the file from the raw bytes.

```
final bytes = result?.files.first.bytes;
final file = File.fromRawPath(bytes!);
```

And I noticed when printing the file it shows the data I intended to read as a string. But it starts with "File: " and has single quotes wrapping the content. So in my case, it worked using the toString method of the File in this hacky way:

`file.toString().substring(7, file.toString().length - 1);`
